doc/release/1.5.0-notes.rst

```diff
@@ -39,39 +57,230 @@ New features
+``np.random.RandomState`` as a seed.
+
+The ``axis`` parameter was added to `scipy.stats.rankdata`. This allows slices 
+of an array along the given axis to be ranked independently.
+
+The ``axis`` parameter was added to `scipy.stats.f_oneway`, allowing it compute 
+multiple one-way ANOVA tests for data stored in n-dimensional arrays. Some
+performance improvements to ``f_oneway`` as well.
+
+The PDF and CDF methods for ``stats.geninvgauss`` are now significantly faster 
+as  the numerical integration to calculate the CDF uses a Cython based 
+``LowLevelCallable``.
+
+Moments of the normal distribution (`scipy.stats.norm`) are now calculated using 
+analytical formulas instead of numerical integration for greater speed and 
+accuracy
+
+Moments and entropy trapezoidal distribution (`scipy.stats.trapz`) are now 
+calculated using analytical formulas instead of numerical integration for 
+greater speed and accuracy
+
+Methods of the truncated normal distribution (`scipy.stats.truncnorm`), 
+especially ``_rvs``, are significantly faster after a complete rewrite.
+
+The `fit` method of the Laplace distribution,  `scipy.stats.laplace`, now uses the
+analytical formulas for the maximum likelihood estimates of the parameters.
+
+Generation of random variates is now thread safe for all SciPy distributions. 
+3rd-party distributions may need to modify the signature of the ``_rvs()`` 
+method to conform to ``_rvs(self, ..., size=None, random_state=None)``. (A 
+one-time VisibleDeprecationWarning is emitted when using non-conformant 
+distributions.)
+
+The Kolmogorov-Smirnov two-sided test statistic distribution 
+(`scipy.stats.kstwo`) was added. Calculates the distribution of the K-S 
+two-sided statistic ``D_n`` for a sample of size n, using a mixture of exact 
+and asymptotic algorithms.
+
+The new function ``median_abs_deviation`` replaces the deprecated 
+``median_absolute_deviation``.
+
+The ``wilcoxon`` function now computes the p-value for Wilcoxon's signed rank 
+test using the exact distribution for inputs up to length 25.  The function has 
+a new ``mode`` parameter to specify how the p-value is to be computed.  The 
+default is ``"auto"``, which uses the exact distribution for inputs up to length 
+25 and the normal approximation for larger inputs.
+
+Added a new cython-based implementation to evaluate guassian kernel estimates,
+which should improve the performance of ``gaussian_kde``
+
+The ``winsorize`` function now has a ``nan_policy`` argument for refined
+handling of ``nan`` input values.
 
 Deprecated features
 ===================
 
 `scipy` deprecations
 --------------------
 
+`scipy.special` changes
+-----------------------
+The ``bdtr``, ``bdtrc``, and ``bdtri`` functions are deprecating non-negative 
+non-integral ``n`` arguments. With this release ``n>0`` is still supported for 
+arbitrary ``n``. However, a deprecation warning is emitted notifying that 
+non-integral n will not be supported with SciPy ``1.7.0`` release.
```

Comment:
I would shorten this to only the first sentence of the paragraph, the rest is fairly redundant.